src/app/people/page.tsx

```diff
@@ -1,5 +1,200 @@
+      const myObserverCallback = (entries: IntersectionObserverEntry[]) => {
+        entries.reverse().forEach(entry => {
+          if (entry.isIntersecting && entry.intersectionRatio > 0) {
+            if (entry.target.id) {
+              setSelected(entry.target.id)
+            }
+          }
+        })
+      }
+
+      const io = new IntersectionObserver(myObserverCallback, { threshold: 0.1 })
+
+      CareerTypes.map(career => {
+        const element = document.querySelector(`#${career.toLowerCase().replaceAll(' ', '-').replaceAll('.', '')}`)
+        if (element) {
+          io.observe(element)
+        }
+      })
+
+      return () => {
+        io.disconnect()
+      }
+    }
+  }, [])
+
+  return (
+    <main>
+      <h1>People</h1>
+      <MainInner>
+        <Sections>
+          {CareerTypes.filter(career => career !== 'Alumni').map(
+            (career, i) =>
+              MEMBERS_KEY.filter(key => MEMBERS[key].career === career && career !== 'Alumni').length >= 1 && (
+                <Section key={career}>
+                  <SectionTitle id={career.toLowerCase().replaceAll(' ', '-').replaceAll('.', '')}>
+                    {career}
+                  </SectionTitle>
+                  <SectionContent>
+                    {MEMBERS_KEY.filter(key => MEMBERS[key].career == career && career != 'Alumni').map(key => (
+                      <MemberCard key={key} mem={MEMBERS[key]} />
+                    ))}
+                  </SectionContent>
+                </Section>
+              )
+          )}
+          <Section key="alumni">
+            <SectionTitle id="alumni">Alumni</SectionTitle>
+            {['Postdoct Researcher', 'Ph.D Student', 'M.S. Student', 'Visiting Researcher', 'Intern'].map(
+              alumniCareer => (
+                <div key={alumniCareer}>
+                  <SubCategoryTitle>{alumniCareer}</SubCategoryTitle>
+                  <AlumniSectionContent>
+                    {MEMBERS_KEY.filter(
+                      key => MEMBERS[key].career === 'Alumni' && MEMBERS[key].affiliation === alumniCareer
+                    ).map(key => (
+                      <AlumniCard key={key} mem={MEMBERS[key]} />
+                    ))}
+                  </AlumniSectionContent>
+                </div>
+              )
+            )}
+          </Section>
+          <SpecialThanksTitle>Special Thanks</SpecialThanksTitle>
+          <SpecialThanksSection>
+            <SpecialThanksImage src="images/jura.jpeg" alt="Jura Coffee Machine" />
+            <SpecialThanksText>
+              <SpecialThanksSubtitle>Jura</SpecialThanksSubtitle>
+              <SpecialThanksDescription>Coffee Machine</SpecialThanksDescription>
+              <SpecialThanksDescription>
+                “Thank you for your steadfast warmth and the delightful brews that kickstart my mornings”
+              </SpecialThanksDescription>
+            </SpecialThanksText>
+          </SpecialThanksSection>
```

Comment:
```suggestion
<Divider />
          <Section key="thanks">
            <SectionTitle id="thanks">Special Thanks</SectionTitle>
            <SpecialThanksSection>
              <SpecialThanksImage src="images/jura.jpeg" alt="Jura Coffee Machine" />
              <SpecialThanksText>
                <SpecialThanksSubtitle>Jura</SpecialThanksSubtitle>
                <SpecialThanksDescription>Coffee Machine</SpecialThanksDescription>
                <SpecialThanksDescription>
                  <i>“Thank you for your steadfast warmth and the delightful brews that kickstart my mornings”</i>
                </SpecialThanksDescription>
              </SpecialThanksText>
            </SpecialThanksSection>
          </Section>
```

For consistency, it would be better to use Section component as much as possible.